addons/Popochiu/Engine/Templates/CharacterTemplate.gd

```diff
@@ -52,3 +55,9 @@ func play_talk() -> void:
 # Use it to play the grab animation for the character
 func play_grab() -> void:
 	.play_grab()
+
+# Updates character position in the current room if anti-glide animation is on
+# should be called every frame in call method track in animation player
+func update_position():
+	E.current_room.update_characters_position(C.PLAYER_2)
```

Comment:
Must be in `PopochiuCharacter.gd`, which is the class from where all characters inherit from. And it should send `self` instead of `C.PLAYER_2`.

This script is only the one used as the template for developers to write their logic.

Reply:
oh, I've already  fixed it but in Goddiu not in template...